### DISCUSSION TITLE
[Documenting] Effort to Document (many if not all) all acronyms used in both PS2, PCSX2 and Discord.

### DISCUSSION THREAD

**Discussion by knight-ryu12** [PCSX2/pcsx2#11464] [General]
It would be nice to have some sort of documenting effort for *ALL*(subjective) acronyms/shorthand.
I can't understand acronyms better than anybody in the discussion, so having documentation/handouts are nice.

**Comment by F0bes** [comment]
Maybe a wiki page on the PCSX2 wiki and or the Github would be suitable for such a thing.
I'll start though:

EE -> Emotion Engine
FPU -> Floating Point Unit
COPx -> Coprocessor x
TLB -> Translation Lookaside Buffer
SPR -> Scratchpad RAM
GIF -> GS InterFace
GS -> Graphics Synthesizer
RT -> Render Target
TEX -> Texture
DMA -> Direct Memory Access
DMAC -> Direct Memory Access Controller
MMIO -> Can be Memory-Mapped IO or Multi Media Instructions
IOP -> Input / Output Processor
SIF -> Serial InterFace
VU0/1 -> Vector Unit 0 / 1
VIF0/1 -> Vector Unit Interface 0 / 1
VPU0/1 -> Vector Processing Unit 0/1
SPU2 -> Sound Processing Unit 2
FIFO -> First In First Out
MADR -> Memory Address
TADR -> Tag Address
SADR -> Scratch Pad Address
QWC -> Quadword Count
CHCR -> Channel Control Register
MFIFO -> Memory FIFO
HLE -> High Level Emulation
LLE -> Low Level Emulation
INTC -> Interrupt Controller
IPU -> Image Processing Unit
SPS -> Spiky Polygon Syndrome
FBMASK -> FrameBuffer Mask
HW -> Hardware
SW -> Software
VM -> Virtual Machine
NVM/NVRAM -> Non Volatile Memory
DEV9 -> Device 9 (Hard Drive and Network Adapter)

**Comment by Mrlinkwii** [comment]
https://github.com/PCSX2/pcsx2/wiki/03-OSD-Legend has a few of them :)

**Comment by Maxkatsur** [comment]
I can't access Discord anynore, because my country has blocked Discord since October this year. Anywhere else to read PCSX2's stable update logs? I don't know.

**Reply by F0bes** [reply]
There has been no stable release report for v2.2.0. If you're ever curious and want to check just go to pcsx2.net and look at the most recent article.

**Comment by chaoticgd** [comment]
What does DVP (in relation to the vector units) stand for? I've been assuming Digital Vector Processor but I'm not entirely sure.
